astroid/nodes/node_classes.py

```diff
@@ -669,7 +718,9 @@ def postinit(
             type_comment_posonlyargs = []
         self.type_comment_posonlyargs = type_comment_posonlyargs
 
-    assigned_stmts: ClassVar[AssignedStmtsCall[Arguments]]
+    assigned_stmts: ClassVar[
+        AssignedStmtsCall[Arguments]
+    ] = protocols.arguments_assigned_stmts
```

Comment:
This is already much better. We could even remove the annotation here?

Reply:
[c644b53](https://github.com/pylint-dev/astroid/pull/2171/commits/c644b53ea04af5e87cdf1ba536823dbe96b29775)